src/app/dashboard/create_budget_form.tsx

```diff
@@ -10,10 +10,15 @@ export function NewBudgetForm({user_id}: {user_id: string}) {
 
   const [name, setName] = useState('Dance Party');
   const [description, setDescription] = useState('Lots of Dancing and Fun');
+  const [location, setLocation] = useState('HSSC N114');
+  const [date, setDate] = useState('2024-MM-DD');
+  const [type, setType] = useState('SEPC');
+  const [cost, setCost] = useState('$500');
```

Comment:
you should remove `cost, setCost` since that shouldn't be set by the user and instead calculated from their items